src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -979,6 +979,7 @@ public static bool IsDefaultExpressionContext(this SyntaxTree syntaxTree, int po
             int position,
             SyntaxToken tokenOnLeftOfPosition,
             CancellationToken cancellationToken,
+            ISet<SyntaxKind> allowedModifiersBefore = null,
             int? allowableIndex = null)
```

Comment:
this parameter is strange.  it's only used to determine if we can offer 'this' or not.  it really should just be 'bool onlyIfFirstParameter' or something like that.  we never actually use any value for it other then 0.